modprobe.d/7640hs_blacklist.conf

```diff
@@ -0,0 +1,12 @@
+blacklist snd_hda_intel
```

Comment:
it seems like there are too many blacklist items, ideally we only want the ones from amd, can you test if it works without the rest?